plugins/tekton/src/components/PipelineRunList/PipelineRunHeader.tsx

```diff
@@ -20,11 +20,13 @@ export const PipelineRunHeader: TableColumn[] = [
     field: 'metadata.name',
     type: 'string',
     render: (row): React.ReactNode => (
-      <ResourceBadge
-        color={tektonGroupColor.value}
-        abbr="PLR"
-        name={(row as PipelineRunKind)?.metadata?.name || ''}
-      />
+      <Link to={`pipelineRun/${(row as PipelineRunKind)?.metadata?.name}`}>
```

Comment:
`(row as PipelineRunKind)?.metadata?.name` id duplicated here, good to keep it in const